packages/client/src/i18n/nl.ts

```diff
@@ -50,12 +50,9 @@ export default {
           "Deze vergunningcheck gaat over kozijnen, deuren, ramen en panelen. U kunt hem gebruiken als u nieuwe plaatst of als u bestaande vernieuwt.",
         "placing bullet": "het plaatsen van de kozijnen zelf.",
       },
-      slopen: {
-        "build without permit exception":
-          "Het gebouw, of het deel van het gebouw wat u gaat slopen, is zonder vergunning gebouwd....?",
-        "intro description":
-          "Deze vergunningcheck gaat over slopen van bouwwerken.",
-        "placing bullet": "het slopen van een bouwwerk zelf.",
+      demolition: {
```

Comment:
All other checkers have a dutch name. I like having an English one here. Maybe add to todo we translate the others too?